src/main/resources/db/migration/V205__add_court_name_cy_to_admin_court_history_table.sql

```diff
@@ -0,0 +1,4 @@
+-- FACT-1500
+-- adds court_name_cy column to admin_court_history table
+
+ALTER TABLE ONLY public.admin_court_history ADD COLUMN IF NOT EXISTS court_name_cy character varying(255);
```

Comment:
We don't need this script, can all be included in the 203 script as flyway hasn't reached the envs yet